src/backend/InvenTree/plugin/templatetags/plugin_extras.py

```diff
@@ -6,7 +6,7 @@
 
 from common.models import InvenTreeSetting
 from common.notifications import storage
-from plugin import registry
+from plugin.registry import registry
```

Comment:
Question: Why this change? The before seems to be no different than the after as we are exposing registry on the plugin module level

Reply:
I think the vscode IDE did not correctly introspect in the first case. So it was just a quality of life fix. Do you want me to revert it?

Reply:
I don’t particularly care either way, just seemed like a nonsensical change